src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -1944,80 +1874,54 @@ namespace HVACMultiSpeedHeatPump {
         // determine the bypass fraction. The simulation converges quickly on mass flow rate. If the zone
         // temperatures float in the deadband, additional iterations are required to converge on mass flow rate.
 
-        // Using/Aliasing
-        using DataSizing::AutoSize;
-        using Fans::GetFanIndex;
-        using Fans::GetFanVolFlow;
-        using FluidProperties::GetDensityGlycol;
-        using FluidProperties::GetSatDensityRefrig;
-
-        using DXCoils::GetDXCoilAvailSchPtr;
-        using PlantUtilities::InitComponentNodes;
-        using PlantUtilities::ScanPlantLoopsForObject;
-        using PlantUtilities::SetComponentFlowRate;
-        using Psychrometrics::PsyRhoAirFnPbTdbW;
-        using ScheduleManager::GetCurrentScheduleValue;
-        using SteamCoils::SimulateSteamCoilComponents;
-        using WaterCoils::GetCoilMaxWaterFlowRate;
-        using WaterCoils::SimulateWaterCoilComponents;
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         static constexpr std::string_view RoutineName("InitMSHeatPump");
-        int InNode;     // Inlet node number in MSHP loop
-        int OutNode;    // Outlet node number in MSHP loop
-        int ZoneInNode; // Zone inlet node number in the controlled zone for MSHP
-        Real64 RhoAir;  // Air density at InNode
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:1968]:(style),[variableScope],The scope of the variable 'ZoneInNode' can be reduced.